src/main/resources/com/databricks/spark/sql/perf/mllib/config/mllib-large.yaml

```diff
@@ -0,0 +1,13 @@
+output: /databricks/spark/sql/mllib-perf-ci
+timeoutSeconds: 1000
```

Comment:
just to confirm, is this timeout good for the large test?

Reply:
It's enough to benchmark 1 estimator, but we'll need to bump it up as we add more.  I can add a comment.